react/react_intro/README.md

```diff
@@ -232,55 +250,56 @@ function App() {
 export default App;
 ```
 
-Every time that you save your application, the browser window will reload the changes that you've made.  You should now see the following screen:
+Every time that you save your application, as long as your server is running, the browser window will reload to reflect the changes that you've made. You should now see the following screen:
 
 ![reactContacts](./images/reactContacts.png)
 
-Now, we can build the `Feed` component.  The feed should have some number of `FeedPost` inside of it.  We can nest components inside of each other to create more complex, dynamic structures.
+## Duplicating Components
```

Comment:
Any chance we can make some part of this section into a more hands-on activity for the learners?